hinglishdistilbertfinetuning.py

```diff
@@ -0,0 +1,302 @@
+import datetime
+import os
+import random
+import time
+
+import numpy as np
+import pandas as pd
+import tensorflow as tf
+import torch
+from IPython.display import clear_output
+from keras.preprocessing.sequence import pad_sequences
+from sklearn import preprocessing
+from sklearn.metrics import accuracy_score, f1_score, precision_recall_fscore_support
+from sklearn.model_selection import train_test_split
+from torch.utils.data import DataLoader, RandomSampler, SequentialSampler, TensorDataset
+from transformers import (
+    AdamW,
+    DistilBertConfig,
+    DistilBertForSequenceClassification,
+    DistilBertTokenizer,
+    get_cosine_with_hard_restarts_schedule_with_warmup,
+    get_linear_schedule_with_warmup,
+)
+
+from hinglishutils import *
+
+
+def hinglishDistilBert(
+    batch_size=8,
+    attention_probs_dropout_prob=0.4,
+    learning_rate=5e-7,
+    adam_epsilon=1e-8,
+    hidden_dropout_prob=0.3,
+    input_name="DistilBert",
```

Comment:
The `input_name` is already in the function name! I don't follow why we need this as an additional input? 

I've a feeling that these should be class objects instead of driver functions?